docs/src/ci-intro.md

```diff
@@ -57,25 +51,34 @@ jobs:
     - uses: actions/checkout@v4
     - uses: actions/setup-node@v4
       with:
-        node-version: 18
+        node-version: lts/*
     - name: Install dependencies
       run: npm ci
     - name: Install Playwright Browsers
       run: npx playwright install --with-deps
     - name: Run Playwright tests
       run: npx playwright test
     - uses: actions/upload-artifact@v4
-      if: ${{ !cancelled() }}
```

Comment:
revert that?

Reply:
I've copied this over from the "on push/pull_request" recipe in `ci.md`. Let's try to use the same snippet between the two - which one do you prefer?

Reply:
`if: ${{ !cancelled() }}` is what we changed it to based on https://github.com/microsoft/playwright/pull/30102

Reply:
hmm, then that missed one of the recipes. I'll update it!

Reply:
done in https://github.com/microsoft/playwright/pull/31988/commits/acbcbf1b472be396429dd8b49d1058eb2b235745

Reply:
I think I got confused here because I copied over from `create-playwright`. Looks like we missed that when updating, here's a PR to also use `!cancelled()` there: https://github.com/microsoft/create-playwright/pull/128